scripts/hostcfgd

```diff
@@ -1322,6 +1401,12 @@ class HostConfigDaemon:
         # Update AAA with the hostname
         self.aaacfg.hostname_update(self.hostname_cache)
 
+        # Update Login messages
+        login_msgs = self.config_db.get_table(
+            swsscommon.CFG_LOGIN_MESSAGE_TABLE_NAME)
+        for key in login_msgs:
+            self.loginmsgcfg.set_login_message(key, login_msgs[key])
```

Comment:
```load``` will save configuration into the cache, so this call to ```set_login_message``` will not set anything because it is the same as in the cache?